### PR TITLE
ciliatedness --> ciliated + add mono and multi ciliated

### LINKED ISSUE #561: ciliatedness --> ciliated + add mono and multi ciliated subclasses
PMID:36924208

### PR DESCRIPTION
**Pull request by dosumis**
Fixes #561

@rays22  - I'd like to get this out ASAP as needed for multiple HCA projects. Could you give it a quick review. Thanks!

### CODE REVIEW

**Comment by dosumis** [comment]
CC @Caroline-99

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -8984,8 +8984,8 @@ is_a: PATO:0002505 ! nucleated
 
 [Term]
 id: PATO:0001408
-name: ciliatedness
-def: "A cellular quality inhering in a bearer by virtue of the bearer's having thin, tail-like projections extending outwards from the cell body." [Wikipedia:http\://en.wikipedia.org/wiki/Ciliated]
+name: ciliated
+def: "A cellular quality inhering in a bearer by virtue of the bearer's having one or more cilia as a part." [PMID:36924208, Wikipedia:http\://en.wikipedia.org/wiki/Ciliated]
```
Comment (rays22) [inline src/ontology/pato-edit.obo:8988]: There is no wikipedia page for ["Ciliated".](https://en.wikipedia.org/wiki/Http:/en.wikipedia.org/wiki/Ciliated) However, there is a <https://en.wikipedia.org/wiki/Cilium> page.

**Review by rays22 — CHANGES_REQUESTED** [review]
Change or drop the wikipedia xref `Wikipedia:http\://en.wikipedia.org/wiki/Ciliated]` for PATO:0001408.

File: src/ontology/pato-edit.obo (hunk 1 above)
Comment (Caroline-99) [reply src/ontology/pato-edit.obo:8988]: I have changed the reference to https://en.wikipedia.org/wiki/Cilium

Comment (rays22) [inline src/ontology/pato-edit.obo:8988]:
```suggestion
def: "A cellular quality inhering in a bearer by virtue of the bearer's having one or more cilia as a part." [PMID:36924208, Wikipedia:http\://en.wikipedia.org/wiki/Cilium]
```

File: src/ontology/pato-edit.obo [hunk 2]
```diff
@@ -8984,8 +8984,8 @@ is_a: PATO:0002505 ! nucleated
 
 [Term]
 id: PATO:0001408
-name: ciliatedness
-def: "A cellular quality inhering in a bearer by virtue of the bearer's having thin, tail-like projections extending outwards from the cell body." [Wikipedia:http\://en.wikipedia.org/wiki/Ciliated]
+name: ciliated
+def: "A cellular quality inhering in a bearer by virtue of the bearer's having one or more cilia as a part." [PMID:36924208, Wikipedia:http\://en.wikipedia.org/wiki/Cilium]
```
Comment (rays22) [inline src/ontology/pato-edit.obo:8988]:
```suggestion
def: "A cellular quality inhering in a bearer by virtue of the bearer's having one or more cilia as a part." [PMID:36924208, wikipedia:Cilium]
```

**Review by rays22 — APPROVED** [review]

**Comment by sbello** [comment]
Nico asked me to look at this today, I know it has been merged but I have a couple comments/questions

- Is there a reason you didn't use the GO definition of cilum (GO:0005929)? 
- Personally circular definitions bug me, so defining ciliated as having cilia without defining cilia feels insufficient to me. I would have used part of the GO definition of cilia in the definition 'specialized eukaryotic organelle that consists of a filiform extrusion of the cell surface and of some cytoplasmic parts' So the definition would be 
   A cellular quality inhering in a bearer by virtue of the bearer's having as a part one or more organelles consisting of a filiform extrusion of the cell surface and of some cytoplasmic parts.

If the term is not meant to define cilia as in GO it would be good to clarify the definition to make this clear.

**Comment by rays22** [comment]
I had taken the CGI (General Class Axiom) associated with [ciliated](http://purl.obolibrary.org/obo/PATO_0001408) into consideration (which does reference GO [cilium](http://purl.obolibrary.org/obo/GO_0005929)) in my review, and I saw no need to request changes in @dosumis 's textual definition: 
image

File: src/ontology/pato-edit.obo [hunk 3]
```diff
@@ -19871,6 +19871,22 @@ is_a: PATO:0000051 ! morphology
 property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0002-7073-9172
 creation_date: 2021-01-23T11:31:53Z
 
+[Term]
+id: PATO:0010007
+name: multiciliated
+def: "A cellular quality inhering in a bearer by virtue of the bearer's having multiple cilia." [PMID:36924208]
+is_a: PATO:0001408 ! ciliated
+property_value: http://purl.org/dc/terms/date "2024-07-23T10:43:20Z" xsd:dateTime
+property_value: http://www.geneontology.org/formats/oboInOwl#http://purl.org/dc/terms/contributor https://orcid.org/0000-0002-7073-9172
```
Comment (anitacaron) [inline src/ontology/pato-edit.obo:19880]: This annotation uses a combination of oio and terms. @Caroline-99, could you please create a PR to fix it? The same happens in the following term: PATO:0010008.
Change the annotation to `http://purl.org/dc/terms/contributor`

**Comment by dosumis** [comment]
@sbello  - as Ray notes - we do have a link to GO - via GCI.  This was a major motivation for me in adding the term.  Not keen to copy and paste text from GO term def as GO term may evolve.

File: src/ontology/pato-edit.obo (hunk 3 above)
Comment (Caroline-99) [reply src/ontology/pato-edit.obo:19880]: Done

**Pull request merged**